src/cli/bunx_command.zig

```diff
@@ -525,8 +525,7 @@ pub const BunxCommand = struct {
                     passthrough,
                     null,
                 );
-                // runBinary is noreturn
-                @compileError("unreachable");
+                Global.exit(if (res.notFailure()) 0 else 1);
```

Comment:
this is expected to exit with the code the subprocess does,
and if the subprocess exits with a signal code instead of an exit code (SIGSEGV, SIGILL), we should raise the same one.

i hope there are already tests for this but you can try to `bunx node -e 'process.abort()'`

Reply:
Makes sense, I updated the ExecResult to include an `exit` method which does the right thing